Updated the SwiftUI docs (#2040)
GetStream/stream-chat-swift: docusaurus/docs/iOS/swiftui/components/message-list.md

)
```

> For link attachments, you can control the link text attributes (font, font weight, color) based on the message. Here's an example how to change the link color based on the message sender, with the `messageLinkDisplayResolver`:

Reviewer: ```suggestion
For link attachments, you can control the link text attributes (font, font weight, color) based on the message. Here's an example of how to change the link color based on the message sender, with the `messageLinkDisplayResolver`:
```

```swift

> let messageDisplayOptions = MessageDisplayOptions(messageLinkDisplayResolver: { message in
>     let color = message.isSentByCurrentUser ? UIColor.red : UIColor.green
>         return [
>             NSAttributedString.Key.foregroundColor: color
>         ]
>     })

Reviewer: ```suggestion
let messageDisplayOptions = MessageDisplayOptions(messageLinkDisplayResolver: { message in
    let color = message.isSentByCurrentUser ? UIColor.red : UIColor.green
    
    return [
        NSAttributedString.Key.foregroundColor: color
    ]
})
```

let messageListConfig = MessageListConfig(messageDisplayOptions: messageDisplayOptions)
let utils = Utils(messageListConfig: messageListConfig)
        
let streamChat = StreamChat(chatClient: chatClient, utils: utils)
```

## Date Indicators

> The SDK supports two types of date indicators - floating overlay and date separators inside the messages. This feature can be configured via the `dateIndicatorPlacement` in the `MessageListConfig`. With the floating overlay option (`.overlay`), the date indicator is shown for a short time whenever a new message appears. On the other hand, if you want to always show the date between messages, similarly to Apple Messages and WhatsApp, you should use the `.messageList` option. You can turn off both options by using the `.none` option. Here's an example how to setup the `messageList` option:

Reviewer: ```suggestion
The SDK supports two types of date indicators - floating overlay and date separators in-between the messages. This feature can be configured via the `dateIndicatorPlacement` in the `MessageListConfig`. With the floating overlay option (`.overlay`), the date indicator is shown for a short time whenever a new message appears. On the other hand, if you want to always show the date between messages, similarly to Apple Messages and WhatsApp, you should use the `.messageList` option. You can turn off both options by using the `.none` option. Here's an example of how to set up the `messageList` option:
```

```swift
let utils = Utils(messageListConfig: MessageListConfig(dateIndicatorPlacement: .messageList))
let streamChat = StreamChat(chatClient: chatClient, utils: utils)
```

> If you want to replace the separating date indicator view, you need to implement the `makeMessageListDateIndicator` method. You can control the size of this view view with the `overlayDateLabelSize` in the `MessageDisplayOptions`.

Reviewer: ```suggestion
If you want to replace the separating date indicator view, you need to implement the `makeMessageListDateIndicator` method. You can control the size of this view with the `overlayDateLabelSize` in the `MessageDisplayOptions`.
```
